Libraries/NavigationExperimental/NavigationAnimatedView.js

```diff
@@ -212,6 +212,7 @@ class NavigationAnimatedView extends React.Component {
     const {renderOverlay} = this.props;
     if (renderOverlay) {
       return renderOverlay(
+        this.props.navigationState,
```

Comment:
Yeah, there are only 2 in 2 of the examples. Unfortunately Flow didn't show any errors. Probably because the examples are not Flow enabled.